src/read_user_input_validate.c

```diff
@@ -38,25 +38,25 @@ void read_user_input_validate(struct passenger *passenger_info, unsigned int tim
         	printf("Invalid passenger id entered");
         	exit(1);
```

Comment:
it shouldnt exit the program when the user enters invalid input. It should ask again. Something like this should work. The rest is fine. 
`int correct = 0;
while( correct == 0){
while((scanf("%d",&arrival_floor))==0){
    	//To check that user enters an integer value only
        	scanf("%*[^\n]");
        	printf("Invalid entry, please enter a valid number: ");
    }
    if(arrival_floor<=0 || arrival_floor > elevator_count){
    if(arrival_floor<=0 || arrival_floor > floor_count){
    		printf("Invalid arrival floor entered!!");
    		correct == 0;
                continue;
    }
 correct = 1;
}`

Reply:
Fixed